src/globals/scss/core/_typography.scss

```diff
@@ -1,435 +1,165 @@
-// $govuk-typography-scale-1: 14;
-// $govuk-typography-scale-2: 16;
-// $govuk-typography-scale-3: 19;
-// $govuk-typography-scale-4: 24;
-// $govuk-typography-scale-5: 27;
-// $govuk-typography-scale-6: 36;
-// $govuk-typography-scale-7: 48;
-// $govuk-typography-scale-8: 80;
-
-@mixin font-smoothing {
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
+@include exports("typography") {
 
-$is-print: false !default;
-
-@mixin _core-font-generator(
-  $font-size: 19px,
-  $font-size-640: 16px,
-  $font-size-print: 14pt,
-  $line-height: (25 / 19),
-  $line-height-640: (20 / 16),
-  $tabular-numbers: false,
-  $font-weight: 400
-  ) {
-  @if $tabular-numbers == true {
-    font-family: $govuk-font-stack-tabular;
-  } @else if $is-print {
-    font-family: $govuk-font-stack-print;
-  } @else {
-    font-family: $govuk-font-stack;
-    @if $govuk-font-stack == "$nta-light" {
-      @if $font-weight > 400 {
-        font-size-adjust: .525;
-      } @else {
-        font-size-adjust: .5;
-      }
-    }
-  }
-  font-weight: $font-weight;
-  text-transform: none;
+  // Headings
 
-  @if $is-print {
-    font-size: $font-size-print;
-    line-height: $line-height;
+  .govuk-heading-xl {
+    @include govuk-font-bold-48;
 
```

Comment:
I believe all of the headings, paragraph classes that use HTML elements need to be normalised. At the moment `<h1 class="govuk-heading-xl">govuk-heading-xl</h1>` will have the default margin-top applied to the `h1`

Reply:
Have pushed a further commit to remove top margins from all headings and paragraphs